### ISSUE TITLE
Route([controller]/[action]) on a controller causes double actions

### ISSUE THREAD

**Issue by haestflod** [haestflod/asp.net-mvc-core-1.0-localized-routing#3]
For account controller at the class declaration:
```
[Route("[controller]/[action]")]
public class AccountController : LocalizationController {
   ...
}
```
Will cause the url to become "account/login/login"  duplicating the action for the route.

If you've for example added this LocalizedRoute:
`[LocalizedRoute("sv", "logga-in", "Logga in")]` 
The route will become: 
`Account/logga-in/Login`

Current workaround:
Don't use `[Route("[controller]/[action]")]` !

**Comment by haestflod** [comment]
After doing unit tests #8 I think the issue lies with LocalizedRoutingConvention.

**Comment by haestflod** [comment]
Seems to be solved in 2.0.2, not sure by what though!
